src/fetcher.ts

```diff
@@ -761,8 +761,9 @@ export default class Fetcher implements CallbackifyInterface {
     if (!this._fetch) {
       throw new Error('No _fetch function available for Fetcher')
     }
-
-    this.appNode = this.store.rdfFactory.blankNode()
+    // This is the name of the graphh we store all the HTTP metadata in
+    this.appNode = this.store.sym('chrome://TheCurrentSession')
```

Comment:
why chrome://?

Reply:
From the call: Comes not from the browser name, but used internally in browsers (predates the Browser Chrome)